### PR TITLE
kebab case

### PR DESCRIPTION
**Pull request by nadinequinr3**
* changes for kebab case updates
* cross-ref added to installation

### CODE REVIEW

File: content/en/platform/corda/5.0-beta/corda-cli-reference/cli-commands/database.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This section lists the Corda CLI `database` arguments. You can use these command
 | <div style="width:160px">Argument</div> | Description                                                                        |
 | --------------------------------------- | ---------------------------------------------------------------------------------- |
 | spec                                    | Generates the database schema from Liquibase.                                      |
-| -c, \-\-clearChangeLog                  | Deletes the changelogCSV in the PWD to force generation of the SQL files.          |
+| -c, \-\-clear-change-log                   | Deletes the changelogCSV in the PWD to force generation of the SQL files.          |
```
Comment (yarivus) [inline content/en/platform/corda/5.0-beta/corda-cli-reference/cli-commands/database.md:17]: should these | chars be aligned on the same column ?

**Review by yarivus — COMMENTED** [review]
see question

File: content/en/platform/corda/5.0-beta/corda-cli-reference/cli-commands/database.md (hunk 1 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-beta/corda-cli-reference/cli-commands/database.md]: The table will be rendered ok (spaces ignored) but I've reformatted it to improve readability.

**Review by AnnaK-R3 — DISMISSED** [review]

**Review by yarivus — COMMENTED** [review]
@nadinequinr3  You have the same table column alignment in the other files

**Review by williamvigorr3 — DISMISSED** [review]
LGTM

**Review by AnnaK-R3 — APPROVED** [review]

**Pull request merged**